lms/templates/learner_dashboard/certificate.underscore

```diff
@@ -0,0 +1,11 @@
+<div class="certificate-container">
+    <p class="title"><%- gettext('XSeries Program Certificates') %>:</p>
+    <div class="certificate-image"></div>
+    <div class="certificate">
+    <% _.each(certificates, function(certificate){ %>
+        <div class="certificate-box">
+            <a class="copy" href="<%- gettext(certificate.credential_url) %>"><%- gettext(certificate.display_name) %></a>
+        </div>
```

Comment:
I don't think these links need wrapper divs. You should be able to set the links to ```display: block;``` and style them accordingly.